sympy/core/tests/test_expr.py

```diff
@@ -1531,6 +1531,10 @@ def test_random():
     assert posify(x)[0]._random() is not None
     assert lucas(n)._random(2, -2, 0, -1, 1) is None
 
+    # issue 8662
+    from sympy import Piecewise, Max
```

Comment:
The import should be at the top of the file, usually. Piecewise is already imported there; you only have to add Max to that list of imports and remove this line.